src/types.ts

```diff
@@ -45,4 +167,13 @@ export class DropDatabaseResponse {
     get message(): string {
         return this._message;
     }
-}
\ No newline at end of file
+}
+
+// Decorator to annotate collection type
+@TigrisCollection("user")
+export class User implements TigrisCollectionType{
```

Comment:
do we need this in the types.ts?

Reply:
no I will remove this and move it to test. good catch.